Yoast/YoastSEO.js
# Stories/1233 raise branch coverage level for get sentence parts

## Summary

This PR can be summarized in the following changelog entry: A test for getSentencePartsSpec.js was added, testing whether German sentences that begin with stopwords are split correctly.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn test`.
* Check that all tests pass.

Fixes #1233
